### ISSUE TITLE
Railroad Track Ears

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7092]
**Preferred term label:**
Railroad Track Ears

**Synonyms**
rail road track ears, rail road ears, railroad track ear deformity, railroad track deformity

**Definition (free text, please give PubMed ID)**

"Railroad track” ears: The top part (curve) of the outer ear is underdeveloped, folded over, and parallel to the curve beneath it, giving the appearance of a railroad track.

https://www.aafp.org/afp/2005/0715/p279.html
https://pubmed.ncbi.nlm.nih.gov/16962173/

**Parent term (use hpo.jax.org/app)**
HP:0040111

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fetal alcohol syndrome 

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by taylordm** [comment]
The synonyms can be expanded as any combination of "railroad", "rail road" etc.

**Comment by pnrobinson** [comment]
I am wondering if there is a synonym for this that does not use the word railroad. I searched a bit and could only find "railroad" synonyms, but the phrase seems slightly derogatory. I will ask some dysmorphologists for help.

**Comment by pnrobinson** [comment]
Thanks to Drs. Hennekam and Firth for advice. This topic appeared in the Elements of Morphology series (https://onlinelibrary.wiley.com/doi/10.1002/ajmg.a.32599, search for the word "railroad" to find the relevant text). The HPO has deferred to Elements for all relevant terms and actually included this comment here
https://hpo.jax.org/app/browse/term/HP:0009897

(which I forgot!). However, it is not easy to find if you start with the phrase "Railroad track ears". We would now consider this to be a deprecated synonym, and I will add it as such to the term but otherwise leave things be (as recommended by Elements of Morphology). Would this work for your use case @taylordm  ?

**Comment by DBHI-BiG** [comment]
Yes, definitely this works, thank you!

**Comment by taylordm** [comment]
I'm ok keeping as is and then using it as a synonym.

**Comment by pnrobinson** [comment]
thanks, done!
